### ISSUE TITLE
hi-ml-multimodal - RSNA pneumonia test-split

### ISSUE THREAD

**Issue by jbdel** [microsoft/hi-ml#744]
Hello and thank you for this good work.

I'm using your pretrained model biovil but its really hard to replicate the zero-shot results from your paper:
Making the Most of Text Semantics to Improve Biomedical Vision–Language Processing

Because the splits are not defined anywhere.
You state

> The evaluation is conducted on Dtest = 9006 images as in [31] (30% eval. / 70% train.) using the dataset’s ground-truth labels.

Is the Dtest the test set of the RSNA Pneumonia Detection Challenge from kaggle ? But there is no test-set labels? Or maybe you sampled from the training set?

If you could link your test-split, that would be very helpful,

Thank you very much in advance
